cellpose/gui/io.py

```diff
@@ -554,6 +554,20 @@ def _masks_to_gui(parent, masks, outlines=None, colors=None):
     else:
         parent.ViewDropDown.setCurrentIndex(0)
 
+def _save_features_csv(parent):
+    """
+    Saves features to CSV if dataset is 2D.
+
+    Args:
+        parent: GUI object with filename and NZ attributes.
+    """
+    filename = parent.filename
```

Comment:
I would recommend an error message or an exception, if there is no file loaded atm.